api/tests/tooling/domain-builder/factory/build-assessment-result.js

```diff
@@ -1,19 +1,24 @@
+const faker = require('faker');
+const { status: assessmentStatuses } = require('../../../../lib/domain/models/AssessmentResult');
 const AssessmentResult = require('../../../../lib/domain/models/AssessmentResult');
+const buildAssessment = require('./build-assessment');
 
 module.exports = function buildAssessmentResult({
   pixScore = 31,
-  status = 'validated',
+  status = faker.random.objectElement(assessmentStatuses),
   emitter = 'PIX-ALGO',
   commentForJury = 'Comment for Jury',
   commentForCandidate = 'Comment for Candidate',
   commentForOrganization = 'Comment for Organization',
-  id = 1,
+  id = faker.random.number(),
   createdAt = new Date('2018-01-12T01:02:03Z'),
   juryId = 1,
-  assessmentId = 2,
+  assessmentId,
```

Comment:
On le builder n'est pas utile
```suggestion 
  assessmentId = faker.random.number(),
```